tests/UI/package.json

```diff
@@ -9,7 +9,7 @@
     "linkchecker": "./node_modules/mocha/bin/mocha campaigns/tools/linkchecker.js",
     "lint-fix": "eslint --fix --ignore-path .gitignore .",
     "lint": "eslint --ignore-path .gitignore .",
-    "specific-test": "./node_modules/mocha/bin/mocha --recursive --file campaigns/utils/setup.js campaigns/$TEST_PATH",
+    "specific-test": "./node_modules/mocha/bin/mocha --bail --recursive --file campaigns/utils/setup.js campaigns/$TEST_PATH",
```

Comment:
Why ?

Reply:
https://mochajs.org/api/mocha#bail => Known as fast fail

Reply:
shouldn't be pushed, nice catch

Reply:
I know what it does, I just didn't understand why it would be necessary, we don't want to fastfail on `specific-test`, we use it quite often and it could be counterproductive to stop on the first fail as we use it to debug

Reply:
should be ok now